saleor/graphql/order/bulk_mutations/order_bulk_create.py

```diff
@@ -0,0 +1,1067 @@
+    )
+    customer_note = graphene.String(description="Note about customer.")
+    notes = NonNullList(
+        OrderBulkCreateNoteInput,
+        description="Notes related to the order.",
+    )
+    language_code = graphene.Argument(
+        LanguageCodeEnum, required=True, description="Order language code."
+    )
+    display_gross_prices = graphene.Boolean(
+        description="Determines whether checkout prices should include taxes, "
+        "when displayed in a storefront.",
+    )
+    weight = WeightScalar(description="Weight of the order in kg.")
+    redirect_url = graphene.String(
+        description="URL of a view, where users should be redirected "
+        "to see the order details.",
+    )
+    lines = NonNullList(
+        OrderBulkCreateOrderLineInput, required=True, description="List of order lines."
+    )
+    delivery_method = graphene.Field(
+        OrderBulkCreateDeliveryMethodInput,
+        required=True,
+        description="The delivery method selected for this order.",
+    )
+    promo_codes = NonNullList(graphene.String, description="List of promo codes.")
+    discounts = NonNullList(OrderDiscountCommonInput, description="List of discounts.")
+    fulfillments = NonNullList(
+        OrderBulkCreateFulfillmentInput, description="Fulfillments of the order."
+    )
+    # TODO fulfillments
+    # TODO invoices = [OrderBulkCreateInvoiceInput!]
+    # TODO transactions: [TransactionCreateInput!]!
+    # TODO discounts (? need to be added/calculated if any ?)
+    # TODO handle order number
+
+
+class OrderBulkCreateResult(graphene.ObjectType):
+    order = graphene.Field(OrderType, description="Order data.")
+    errors = NonNullList(
+        OrderBulkCreateError,
+        description="List of errors occurred on create attempt.",
+    )
+
+
+class OrderBulkCreate(BaseMutation, I18nMixin):
+    count = graphene.Int(
+        required=True,
+        default_value=0,
+        description="Returns how many objects were created.",
+    )
+    results = NonNullList(
+        OrderBulkCreateResult,
+        required=True,
+        default_value=[],
+        description="List of the created orders.",
+    )
+
+    class Arguments:
+        orders = NonNullList(
+            OrderBulkCreateInput,
+            required=True,
+            description=f"Input list of orders to create. Orders limit: {MAX_ORDERS}.",
+        )
+        error_policy = ErrorPolicyEnum(
+            required=False,
+            default_value=ErrorPolicyEnum.REJECT_EVERYTHING.value,
+            description=(
+                "Policies of error handling. DEFAULT: "
+                + ErrorPolicyEnum.REJECT_EVERYTHING.name
+            ),
+        )
+
+    class Meta:
+        description = "Creates multiple orders."
+        permissions = (OrderPermissions.MANAGE_ORDERS_IMPORT,)
+        error_type_class = OrderBulkCreateError
+        error_type_field = "bulk_order_errors"
```

Comment:
We could skip this field. This field is required to add deprecated errors to the mutation.